### DISCUSSION TITLE
Does react-router strictly require the other dependencies in the tutorial?

### DISCUSSION THREAD

**Discussion by julianeden** [remix-run/react-router#9808] [Q&A]
The Getting Started page instructs the use of `npm install react-router-dom localforage match-sorter sort-by`.

Are those other 3 packages mandatory?

**Comment by arift** [comment]
According to [this](https://github.com/remix-run/react-router/issues/11099), no it doesn't. This should be made more clear in the documentation so people don't end up installing dependencies they don't need.
